src/modScaleThumbnails.js

```diff
@@ -8,19 +8,19 @@ var Mod = class extends mod.Base {
         // Thumbnails on main monitor
         this.bkp_MAX_THUMBNAIL_SCALE = workspaceThumbnail.MAX_THUMBNAIL_SCALE
         workspaceThumbnail.MAX_THUMBNAIL_SCALE = 0.1
-
+        
         // Thumbnails on second monitor
+        if (!SecondaryMonitorDisplay) return;
```

Comment:
Looks great!

May I also ask you to remove all other changes except this line? (as no other changes actually change anything).

Thank you